src/utils.js

```diff
@@ -90,13 +87,13 @@ export async function getTableData(indicators,selectCode,selectItem){
 	return selectItem
 }
 
-export async function getNomis(url, geographicCodesStore, indicatorCode) {
+export async function getNomis(url, dataService, geographicCodesStore, selectedCategoryTotals, indicatorCode) {
 	let geoCodesStore = get(geographicCodesStore)
 	if (geoCodesStore.length == 0) {
-		let geoCodes = await localDataService.getGeographicCodes(url)
+		let geoCodes = await dataService.getGeographicCodes(url)
```

Comment:
so, this looks like its getting the geocodes as a seperate call to getting the rest of the data?

Reply:
yes, the current structure is a bit of a scaffold because we're still using CSVs.  atm we have separate getGeographicCodes and getCategoryTotals methods, but eventually we'll be integrating those processes within a single API call